docs/internal_api.yml

```diff
@@ -512,6 +512,43 @@ paths:
         500:
           $ref: "#/responses/InternalServerError"
 
+  /tenants/{tenant_id}/device/deployments/last:
+    post:
+      operationId: Get last device deployment status
+      tags:
+        - Internal API
+      summary: Get status of the last device devployment
+      description: |
+        Return the status of the last unsucessful device deployment.
+      parameters:
+        - name: tenant_id
+          in: path
+          description: Tenant identifier.
+          required: true
+          type: string
+        - name: devicesIds
+          in: body
+          description: An array of device ids for which to return the statuses.
+          type: array
+          items:
+            type: string
```

Comment:
```suggestion
          schema:
            type: array
            items:
              type: string
```

Reply:
I'm also wondering if the list shouldn't be named (object with a list); this way it will be possible to extend it in the future if needed

Reply:
fixed, together with schema and the wrapping object.